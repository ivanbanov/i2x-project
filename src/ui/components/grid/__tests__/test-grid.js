import React from 'react';
import renderer from 'react-test-renderer';
import Grid from '../';

jest.mock('../styles.styl', () => ({}));

describe('Grid', () => {
  it('should render properly', () => {
    const component1 = renderer.create(
      <Grid>
        <div>Grid</div>
      </Grid>
    );

    const component2 = renderer.create(
      <Grid fluid>
        <div>Grid</div>
      </Grid>
    );

    const tree1 = component1.toJSON();
    const tree2 = component2.toJSON();

    expect(tree1).toMatchSnapshot();
    expect(tree2).toMatchSnapshot();
  });
});
