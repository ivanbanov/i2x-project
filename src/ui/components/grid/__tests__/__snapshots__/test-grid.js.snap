// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid should render properly 1`] = `
<div
  className=""
>
  <div>
    Grid
  </div>
</div>
`;

exports[`Grid should render properly 2`] = `
<div
  className="fluid"
>
  <div>
    Grid
  </div>
</div>
`;
